charts/spire/charts/spire-server/templates/tornjak-config.tpl

```diff
@@ -0,0 +1,38 @@
+{{- if eq (.Values.tornjak.enabled | toString) "true" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tornjak-config
+  namespace: {{ .Values.namespace }}
```

Comment:
Should use the namespace macro, since it contains more override logic.

Reply:
My bad, I missed that one....